indent-lint.el

```diff
@@ -87,8 +87,12 @@ Function will be called with 2 variables; `(,raw-buffer ,indent-buffer)."
          (with-temp-file ,dest-file
            (insert-file-contents ,src-file)
            (let ((buffer-file-name ,(buffer-name buf)))
-             (normal-mode)
              (funcall #',(with-current-buffer buf major-mode))
+             (dolist (argcell ',(with-current-buffer buf (buffer-local-variables)))
+               (let ((arg (car argcell))
+                     (val (cdr argcell)))
+                 (ignore-errors
+                     (set arg val))))
```

Comment:
I would use `setq-local` instead of `set` to avoid changing the global value of each variable.

Reply:
Yes. However, the Emacs exits immediately after executing indent-region.
So we don't need to `make-local-variable`. The status of Emacs started by async.el has no effect on interactive Emacs.

If we try to use `setq-local`, it will need to use a dirty `eval` because it is macro.
It's hard to explain why...